call-for-submissions.md

```diff
@@ -61,25 +65,27 @@ We prioritize the workers' perspective and working people representing themselve
 
 Here are some examples of content we typically reject:
 
-- Critiques of tech imperialism, capitalism, injustice, etc. &mdash; or proposals for alternatives &mdash; that are not based on your lived experience.
-- Abstract or niche commentaries that lack a universal theme like work, working conditions, or solidarity beyond the workplace.
-- Narratives from corporate nonprofit staff or campaign consultants, not from workers.
-- Anything that reads like a press release or impersonal op-ed.
-- General commentary on labor or the economy that doesn’t relate to the tech industry.
+- Impersonal op-eds that are not based on lived experience.
+- Broad, sweeping critiques of tech imperialism, capitalism, injustice, etc.
+- Press releases from campaign consultants or corporate nonprofit staff who are not rank-and-file workers.
+- Narratives about projects that are more about the solution than the struggle of organizaing or collective action.
+- Niche commentaries that lack a universal theme like work, working conditions, or solidarity beyond the workplace.
 
 ### How to submit
 
 <mark>Here’s how to start the process:</mark>
 
+1. Browse <a href="/archive/">our past issues</a> for inspiration.
 1. Email us your rough ideas or pitches to [twcnewsletter@protonmail.com](mailto:twcnewsletter@protonmail.com)!
-1. We’ll reach out to start a conversation.
-1. Then, we’ll help you get a first draft together, usually in a call.
+1. We’ll reach out to have a two-way conversation to help you get a first draft together.
```

Comment:
Conversations are, by definition, multi-way. Seems redundant to talk about a "two-way" conversation (and also inaccurate, since likely contributors will talk to more than one editor).